docs/src/tutorial/custom-data.rst

```diff
@@ -0,0 +1,162 @@
+Run the workflow using custom data
+==================================
+
+In this tutorial, you will run the workflow using custom focal data in addition to the example reference data. The reference data will serve as background context for the new data.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`example-data`. These instructions will set up the command line environment used in this tutorial.
+
+Setup
+-----
+
+If you are not already there, change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+Curate data from GISAID
+-----------------------
+
+.. warning::
+
+   These steps require a GISAID account. `Register <https://www.gisaid.org/registration/register/>`__ if you do not have an account yet. However, registration may take a few days. Follow `alternative data preparation methods <../guides/data-prep.html>`__ if you wish to continue this tutorial in the meantime.
+
+We will retrieve 10 sequences from GISAID's EpiCoV database.
+
+1. Navigate to `GISAID <https://www.gisaid.org/>`__ and select **Login**.
+
+   .. image:: ../images/gisaid-homepage.png
+      :width: 400
+      :alt: GISAID login link
+
+2. Login to your GISAID account.
+
+   .. image:: ../images/gisaid-login.png
+      :width: 200
+      :alt: GISAID login
+
+3. In the top left navigation bar, select **EpiCoV** then **Search**.
+
+   .. image:: ../images/gisaid-epicov-search.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+4. Select the first 10 records.
+
+   .. note::
+
+      You can filter and select a different set of records here, but note that more records will take the workflow longer to run, and the GISAID web interface has a maximum of 5,000 records per download.
+
+   .. image:: ../images/gisaid-select-records.png
+      :width: 700
+      :alt: GISAID EpiCoV Search
+
+5. Select **Download** in the bottom right of the search results.
+6. Select **Input for the Augur pipeline** as the download format.
+7. Select **Download**.
+
+   .. image:: ../images/gisaid-augur-pipeline-download.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+   .. note::
+
+      You may see different download options, but it is fine as long as **Input for the Augur pipeline** is available.
+
+8. Extract by opening the downloaded ``.tar`` file in your file explorer. It contains two files: one ending with ``.metadata.tsv`` and another with ``.sequences.fasta``.
```

Comment:
What if we ask users to download the tar file into the `ncov/data/` directory and extract it there? Maybe I'm too used to command line file operations, but opening two separate Finder windows to copy from `~/Downloads/` to `~/ncov/data/` felt like too many extra steps.

Reply:
I want to keep this tutorial cross-platform. If we provide shell-specific commands such as `curl`/`wget`, or even line breaks using `\`, it won't work for Windows PowerShell users.

For the download step, we could summarize the intent and nest the drag-drop instructions, like:

> 8. Extract the files to the `data/` directory in the `ncov/` directory:
>     1. Extract by opening the downloaded ``.tar`` file in your file explorer. It contains two files: one ending with ``.metadata.tsv`` and another with ``.sequences.fasta``.
>     2. Move these extracted files to the `data/` directory in the `ncov/` directory.
> 9. Rename the files as ``custom.metadata.tsv`` and ``custom.sequences.fasta``.

But that seems a bit verbose to me.

Reply:
Oh yeah, I wasn't thinking to provide platform-specific instructions, but just to say "Download into `ncov/data/`" as the default behavior, just to make life easier across platforms instead of downloading wherever (e.g., not into `~/Downloads/` by default).

Reply:
I have my browser set to automatically download to `~/Downloads/` without prompt, and I imagine some readers will have the same as well.

With ad130748 I've re-ordered and re-worded a bit so the steps should accommodate all behaviors.